config/locales/en.yml

```diff
@@ -58,3 +58,18 @@ en:
       sign-up: "Sign up"
       text1: "Sign up"
       text2: "This will be a signup page for new users."
+      welcome_user: "Welcome to Sample Application!"
+      name: "Name"
+      pass: "Password"
+      confirm: "Confirmation"
+      btn: "Create my account"
+    create:
+      welcome_user: "Welcome to the Sample App!"
+    show:
+      danger: "User not exists!"
+  shared:
+    error_messages:
+      content: "The form contain"
```

Comment:
trong cùng 1 câu chữ thì không phân tách e nha, vì vị trí từ ngữ có thể khác nhau tùy quốc gia.
Em define chung 1 câu i18n thôi nhé:
count:
one: "The form contain %{count} error"
other: "The form contain %{count} errors"

Reply:
vâng chị ^^